engine/timer.cpp

```diff
@@ -12,6 +12,12 @@ namespace pong
 
 float Timer::frameTime = 1.0f;
 
+void Timer::Init()
+{
+    const auto currentTime = std::chrono::system_clock::now();
+    Timer::frameTime = (std::chrono::duration_cast<std::chrono::duration<float>>(currentTime - mLastTime)).count();
```

Comment:
nice adjustment to chrono

you could save yourself some brain cells and fingers by doing a `using std::chrono` here, seeing as the std library just LOVES its nested namespace and insane templates

Reply:
Yeah writing in that timer had some long lines to it haha. I just wasn't sure how recommended it is to use `using` in general. I know ideally you at least don't use `using std`

Reply:
There are some reasons to use it, one valid one is in the context of a single function when you have extreme use of some really long class name/namespace that you want to pair down and make less egregious, like this.

I don't love file wide uses of it, or uses that just shave off a couple characters. In general, you should write your code so it's easily READ not easily WRITTEN. You read code a billion times, and hopefully only write it once. A little over verbosity is good